azure/README.md

```diff
@@ -0,0 +1,197 @@
+
+Your AKS cluster should be visible in your resource group. Want a different region? You can modify `terraform.tfvars` or input it directly using the `region` variable in plan/apply.
+
+Are you done playing? Please run `terraform destroy` twice to clean up.
+
+### Test it
+
+When you have completed the installation steps, you can do `kubectl port-forward service/wrongsecrets-balancer 3000:3000` and then go to [http://localhost:3000](http://localhost:3000).
+
+Want to know how well your cluster is holding up? Check with
+
+```sh
+    kubectl top nodes
+    kubectl top pods
+```
+
+### Configuring CTFd
+
+You can use the [Juiceshop CTF CLI](https://github.com/juice-shop/juice-shop-ctf) to generate CTFd configuration files.
+
+Follow the following steps:
+
+```shell
+    npm install -g juice-shop-ctf-cli@9.1.2
+    juice-shop-ctf #choose ctfd and https://wrongsecrets-ctf.herokuapp.com as domain. No trailing slash! The key is 'test', by default feel free to enable hints. We do not support snippets or links/urls to code or hints.
+```
+
+Now visit the CTFd instance and setup your CTF. If you haven't set up a load balancer/ingress, the you can use `kubectl port-forward -n ctfd $(kubectl get pods --namespace ctfd -l "app.kubernetes.io/name=ctfd,app.kubernetes.io/instance=ctfd" -o jsonpath="{.items[0].metadata.name}") 8000:8000` and go to `localhost:8000` to visit CTFd.
+
+_!!NOTE:_ **The following can be dangerous if you use CTFd `>= 3.5.0` with wrongsecrets `< 1.5.11`. Check the `challenges.json` and make sure it's 1-indexed - a 0-indexed file will break CTFd!** _/NOTE!!_
+
+Then use the administrative backup function to import the zipfile you created with the juice-shop-ctf command.
+After that you will still need to override the flags with their actual values if you do use the 2-domain configuration. For a guide on how to do this see the 2-domain setup steps in the general [README](../readme.md)
+Want to setup your own? You can! Watch out for people finding your key though, so secure it properly: make sure the running container with the actual ctf-key is not exposed to the audience, similar to our heroku container.
+
+Want to make the CTFD instance look pretty? Include the fragment located at [./k8s/ctfd_resources/index_fragment.html](/k8s/ctfd_resources/index_fragment.html) in your index.html via the admin panel.
+
+### Clean it up
+
+When you're done:
+
+1. Kill the port forward.
+2. Run `terraform destroy` to clean up the infrastructure. Note that you may need to repeat the destroy to fully clean up.
+3. If you've used the shared state, `cd` to the `shared-state` folder and run `terraform destroy` there too.
+4. Run `rm terraform.tf*` to remove local state files.
+
+### A few things to consider
+
+1. Does your worker node now have access as well?
+2. Can you easily obtain the AKS managed identity of the Node?
+3. Can you get the secrets in the Key vault? Which paths do you see?
```

Comment:
this might need a little more description ;) in AWSfolder  as well maybe?

Reply:
@bendehaan Can you support on this

Reply:
We initially chose to add these questions to get the organizer thinking in wrongsecrets and the AWS setup. We could remove them, but I don't think it's worth it to do a deep dive here.